funnel/views/event.py

```diff
@@ -7,12 +8,20 @@
 from .. import app, lastuser
 from ..models import (db, Profile, ProposalSpace, ProposalSpaceRedirect, Participant, Event, Attendee, SyncTicket, ContactExchange)
 import baseframe.forms as forms
-from ..forms import ParticipantForm, ParticipantBadgeForm
+from ..forms import ParticipantForm, ParticipantBadgeForm, ParticipantImportForm
 from helpers import split_name, format_twitter, make_qrcode
 from sqlalchemy.exc import IntegrityError
 from ..extapi.explara import ExplaraAPI
 
 
+def lowercase_keys(dictiter):
+    return [dict((k.lower(), v) for k, v in d.iteritems()) for d in dictiter]
+
+
+def csvstring_to_rows(stream):
+    return lowercase_keys(unicodecsv.DictReader([row for row in StringIO.StringIO(stream)]))
```

Comment:
This is a bit convoluted. You've already got a stream. Why are you putting it through StringIO again?